### DISCUSSION TITLE
Valhalla Python packages on PyPI

### DISCUSSION THREAD

**Discussion by nilsnolde** [valhalla/valhalla#3563] [Show and tell]
after a loooong time fiddling with this, we're proud to finally have the workflow standing to publish valhalla bindings on PyPI for all major platforms & not-too-ancient linux distros and all currently active **c**python versions (3.7 - 3.10)

enjoy and pls let us know if there are problems in the repo's issue tracker!

repo: https://github.com/gis-ops/pyvalhalla
pypi: https://pypi.org/project/pyvalhalla

**Comment by kevinkreiser** [comment]
This is awesome! Ihr Jungs haben eine ausgezeichnete Arbeit geleistet @nilsnolde @TimMcCauley @chrstnbwnkl
